actioncable/Rakefile

```diff
@@ -1,15 +1,14 @@
 require 'rake/testtask'
 require 'pathname'
-require 'sprockets'
-require 'coffee-script'
 require 'action_cable'
+require 'blade'
 
 dir = File.dirname(__FILE__)
 
 task :default => :test
 
 task :package => "assets:compile"
-task "package:clean" => "assets:clean"
+task "package:clean"
```

Comment:
Don't we still we need to define this task, and provide a way for the compiled file to be deleted? 😬